docker-compose.yml

```
@@ -0,0 +1,13 @@
+version: '3'
+
+services:
+    maven:
+        image: maven:3.3-jdk-8
```

You'll need to update this to
```suggestion
        image: maven:3.6-jdk-11
```
now, since I updated the maven config to target building with jdk 11: https://github.com/smrealms/TheLazyTrader/blob/ede4fc63ccefa58cd48aaac0e79f782f16d0fa4c/.github/workflows/build.yml